### PR TITLE
migrate registry in k8s.io subproject from k8s.gcr.io to registry.k8s.io

### PR DESCRIPTION
**Pull request by rayowang**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it:**
This PR will update the uses of deprecating k8s.gcr.io registry to registry.k8s.io

**Which issue(s) this PR fixes:**

Fixes # https://github.com/kubernetes/k8s.io/issues/4738

**Special notes for your reviewer:**
Search results: https://cs.k8s.io/?q=k8s.gcr.io&i=nope&files=&excludeFiles=vendor%2F&repos=kubernetes/k8s.io

**Does this PR introduce a user-facing change?:**

NONE

### CODE REVIEW

File: apps/gcsweb/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ need a documented link to browse.
 `gcsweb` is built from source code at
 https://github.com/kubernetes/test-infra/tree/master/gcsweb into Docker
 container, which is then uploaded to private container storage at
-https://k8s.gcr.io/gcsweb-amd64 and fetched during processing
+https://registry.k8s.io/gcsweb-amd64 and fetched during processing
```
Comment (ameukam) [inline apps/gcsweb/README.md:33]:
```suggestion
registry.k8s.io/gcsweb-amd64 and fetched during processing
```

**Comment by rayowang** [comment]
@ameukam Hello, I am doing some work related to kubernetes recently, so I want to learn more about the project by solving some issues. But I found that because I am not a member, I cannot set ok-to-test for CI automated testing after submitting the PR, which often blocks my progress and reduces my efficiency.

I read the requirements for membership application, and I basically meet the requirements.
but according to the organization's application for membership, I need to communicate in advance with a sponsor associated with my community contribution. Thank you for your help in the past, I hope you will agree to sponsor my membership application, can you help me?

**Comment by mrbobbytables** [comment]
@rayowang I cannot sponsor =/ chaodaiG and I work for the same company (Google), sponsors have to be from different organizations.

**Pull request merged**